check-application-health-after-outage.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Check application health after an outage
+
+description: Describes aspects of your application you can check to ensure they're up and running after an outage.
+
+keywords: openshift, devops, outage, recovery
+
+page purpose: Gives guidelines on what to check on in an application if there has a platform-wide outage.
+
+audience: developer
+
+author: Jonathan Bond
+
+content owner: Cailey Jones
+---
+
+# Post-Outage Application Checklist
+
+One of the benefits of the OpenShift platform is a greatly reduced chance of significant unexpected outages and the near-elimination of scheduled outages. The platform is adaptable and recoverable in a way that legacy architecture is not and this lets the platform recover from hardware issues invisibly in many cases.
+
+Still, it's always possible that an outage affects the platform and brings down the applications hosted there.
+
+If this happens, speedy recovery is important. Use these guidelines to ensure your application recovers quickly and effectively.
+
+## Check reporting channels
+
+If there is a platform-wide outage, check the following sources for more information:
+
+- RocketChat channels `#devops-alerts`, `#devops-sos` and `#devops-operations`
+
+- Off-site [status page](https://status.developer.gov.bc.ca), if you can't access RocketChat
+
+The Platform Services team keeps the community informed about the status of the outage. While the outage is in progress, there isn't much the team can do. Review the status and stay ready for the outage to end.
+
+## Check your application
+
+When the outage recovers, it's likely that everything recovers on its own, due to the resiliency of the applications.
+
+However, you can check the following to make sure everything is working as expected:
+
+### Connect to your application
+
+Connect to your application to see if it's up and running. If an initial check of the functionality shows that it's working as intended, then there's nothing more for you to do, though you can continue further checks if you want.
+
+### Check your pods
+
+Your pods will likely scale back up on their own. However, it's still a good idea to ensure that this has occurred as expected.
+
+- If it hasn't, this may indicate a problem with your deployment-config or with your health checks.
+
+- If you're having problems connecting to your application and your pods look healthy, restart them. If that works, that indicates that you may need to build a more complex and robust health check for those pods.
+
+#### Crash Loop Backoff
+This error usually indicates a problem with your application: it's repeatedly attempting to boot and failing.
+
+Your application logs should have more information and may provide further information about how to fix this issue.
```

Comment:
```suggestion
Your application logs should have more information and may provide further information about how to fix this issue. If your application enters CrashLoopBackoff during recovery from an outage, this may indicate that your application is not designed to be fully resilient.
```